modules/private-shared-lib/src/types/bulk_transaction_entity_repo.ts

```diff
@@ -24,7 +24,8 @@
 
 'use strict';
 
-import { BulkTransactionState, IEntityStateRepository, IndividualTransferState } from '@module-domain';
+import { IEntityStateRepository } from '..';
+import { BulkBatchState, BulkTransactionState, IndividualTransferState } from '../domain';
 
 export type IBulkTransactionEntityRepo = {
```

Comment:
Technically this should be Interface, and not a Type (although they are technically the same thing 🤯 )